partiql-conformance-tests/build.rs

```diff
@@ -46,6 +46,18 @@ fn main() -> io::Result<()> {
             .unwrap_or_else(|error| panic!("Failure when writing to file: {:?}", error));
     }
 
-    dir_to_mods(tests_path);
+    let sub_tests_dir = "partiql_tests";
+    let sub_tests_path = Path::new(sub_tests_dir);
+    dir_to_mods(&tests_path.join(sub_tests_path));
+    File::create(&tests_path.join("mod.rs"))
+        .expect("mod.rs created in root test folder")
+        .write_all(
+            format!(
+                "#[cfg(feature = \"conformance_test\")]\n#[rustfmt::skip]\nmod {};\n",
```

Comment:
Adds the "conformance_test" flag to the top-level `mod.rs` file in the `tests/` directory. 

And also skips the rustfmt checks for the generated files. `rustfmt` was complaining in a previous [GH Actions run](https://github.com/partiql/partiql-lang-rust/runs/6183724158?check_suite_focus=true#step:8:47) about the order of modules in `mod.rs` and about reformatting longer lines.

Reply:
Perhaps this is also comment worthy in the code.

Reply:
Or we should run `rustfmt` on the generated code

Reply:
Changed to run `rustfmt` on the generated code.